src/tunnel.rs

```diff
@@ -410,17 +412,27 @@ make_tunnel_type! {
 
 pub async fn forward(id: &String, addr: String) -> PyResult<()> {
     let tun = &get_storage_by_id(id).await?.tunnel;
+
+    // You can force a tunnel to be a pipe by prepending "pipe:" to the address.
     let is_pipe =
-        addr.starts_with(PIPE_PREFIX) || addr.starts_with("unix:") || addr.starts_with("tun-");
+        addr.starts_with(PIPE_PREFIX) || addr.starts_with(UNIX_PREFIX) || addr.contains("/");
 
     let res = if is_pipe {
         let mut tun_addr = addr.clone();
+        // Remove any prefix text before "/"
```

Comment:
`pipe:` and `unix:` are removed below, what is this meant to handle?

Reply:
Oh woops, my bad. I thought we were using "/" as a prefix delimiter. 

You meant to just use it as an indicator that it is a pipe.

Reply:
Ah, yes! Just using `/` as an indicator. Want `./moo.sock`, `sockets/moo.sock` and `/tmp/moo.sock` to all work and not be modified. Granted we might have to fight more on relative filenames with the upcoming rust changes, but can deal with that when the new rust version is adopted.